target/linux/bcm27xx/bcm2708/config-6.6

```diff
@@ -143,7 +143,6 @@ CONFIG_EDAC_SUPPORT=y
 CONFIG_EXCLUSIVE_SYSTEM_RAM=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
```

Comment:
Setting `CONFIG_EXT4_FS_POSIX_ACL` should not be needed any more. I think these are not small targets and `CONFIG_EXT4_FS_POSIX_ACL` should be set because `USE_FS_ACL_ATTR` is set now. 

Maybe your configuration is not updated correctly, please check with `./scripts/diffconfig.sh`  is these settings are set to the default value.

Reply:
You can remove all the `*FS_POSIX_ACL=y` options for the kernel configuration files and only keep the options deactivated in the generic kernel configuration.

Reply:
> You can remove all the `*FS_POSIX_ACL=y` options for the kernel configuration files and only keep the options deactivated in the generic kernel configuration.

done all `*FS_POSIX_ACL=y` removed.



> Maybe your configuration is not updated correctly, please check with `./scripts/diffconfig.sh` is these settings are set to the default value.

On ath79 generic returned to the default, value not on tiny, but also happen without applying the patch all `procd-ujail` are missed.
diff --side
```
CONFIG_ALL_KMODS=y                                              CONFIG_ALL_KMODS=y
CONFIG_ALL_NONSHARED=y                                          CONFIG_ALL_NONSHARED=y
CONFIG_DEVEL=y                                                  CONFIG_DEVEL=y
CONFIG_TARGET_PER_DEVICE_ROOTFS=y                               CONFIG_TARGET_PER_DEVICE_ROOTFS=y
CONFIG_AUTOREMOVE=y                                             CONFIG_AUTOREMOVE=y
CONFIG_BPF_TOOLCHAIN_BUILD_LLVM=y                               CONFIG_BPF_TOOLCHAIN_BUILD_LLVM=y
# CONFIG_BPF_TOOLCHAIN_NONE is not set                          # CONFIG_BPF_TOOLCHAIN_NONE is not set
CONFIG_BUILDBOT=y                                               CONFIG_BUILDBOT=y
CONFIG_COLLECT_KERNEL_DEBUG=y                                   CONFIG_COLLECT_KERNEL_DEBUG=y
CONFIG_DEFAULT_procd-ujail=y                                  <
CONFIG_HAS_BPF_TOOLCHAIN=y                                      CONFIG_HAS_BPF_TOOLCHAIN=y
CONFIG_IB=y                                                     CONFIG_IB=y
CONFIG_JSON_CYCLONEDX_SBOM=y                                    CONFIG_JSON_CYCLONEDX_SBOM=y
CONFIG_KERNEL_BUILD_DOMAIN="buildhost"                          CONFIG_KERNEL_BUILD_DOMAIN="buildhost"
CONFIG_KERNEL_BUILD_USER="builder"                              CONFIG_KERNEL_BUILD_USER="builder"
CONFIG_KERNEL_DEBUG_INFO=y                                      CONFIG_KERNEL_DEBUG_INFO=y
CONFIG_KERNEL_DEBUG_INFO_REDUCED=y                              CONFIG_KERNEL_DEBUG_INFO_REDUCED=y
CONFIG_KERNEL_DEBUG_KERNEL=y                                    CONFIG_KERNEL_DEBUG_KERNEL=y
CONFIG_KERNEL_KALLSYMS=y                                        CONFIG_KERNEL_KALLSYMS=y
CONFIG_MAKE_TOOLCHAIN=y                                         CONFIG_MAKE_TOOLCHAIN=y
CONFIG_MODULE_DEFAULT_procd-ujail=y                           <
CONFIG_PACKAGE_libbpf=m                                         CONFIG_PACKAGE_libbpf=m
CONFIG_PACKAGE_libelf=m                                         CONFIG_PACKAGE_libelf=m
CONFIG_PACKAGE_libpcap=m                                        CONFIG_PACKAGE_libpcap=m
CONFIG_PACKAGE_libxdp=m                                         CONFIG_PACKAGE_libxdp=m
CONFIG_PACKAGE_qosify=m                                         CONFIG_PACKAGE_qosify=m
CONFIG_PACKAGE_tc-tiny=m                                        CONFIG_PACKAGE_tc-tiny=m
CONFIG_PACKAGE_xdp-filter=m                                     CONFIG_PACKAGE_xdp-filter=m
CONFIG_PACKAGE_xdp-loader=m                                     CONFIG_PACKAGE_xdp-loader=m
CONFIG_PACKAGE_xdpdump=m                                        CONFIG_PACKAGE_xdpdump=m
CONFIG_PACKAGE_zlib=m                                           CONFIG_PACKAGE_zlib=m
# CONFIG_PKG_ASLR_PIE_NONE is not set                           # CONFIG_PKG_ASLR_PIE_NONE is not set
CONFIG_PKG_ASLR_PIE_REGULAR=y                                   CONFIG_PKG_ASLR_PIE_REGULAR=y
CONFIG_REPRODUCIBLE_DEBUG_INFO=y                                CONFIG_REPRODUCIBLE_DEBUG_INFO=y
CONFIG_SDK=y                                                    CONFIG_SDK=y
CONFIG_SDK_LLVM_BPF=y                                           CONFIG_SDK_LLVM_BPF=y
CONFIG_TARGET_ALL_PROFILES=y                                    CONFIG_TARGET_ALL_PROFILES=y
CONFIG_USE_LLVM_BUILD=y                                         CONFIG_USE_LLVM_BUILD=y
CONFIG_DEFAULT_procd-ujail=y                                  <
CONFIG_MODULE_DEFAULT_procd-ujail=y                           <
# CONFIG_PACKAGE_procd-ujail is not set                       <
```